gcloud-java-resourcemanager/src/main/java/com/google/gcloud/resourcemanager/testing/LocalResourceManagerHelper.java

```diff
@@ -0,0 +1,588 @@
+package com.google.gcloud.resourcemanager.testing;
```

Comment:
file contains many trailing whitespaces.
see [here](http://stackoverflow.com/questions/1043433/how-to-auto-remove-trailing-whitespace-in-eclipse) for fixiing suggestions.